code/datums/diseases/_MobProcs.dm

```diff
@@ -127,7 +127,7 @@
 					Cl = M.wear_mask
 					passed = prob((Cl.permeability_coefficient*100) - 1)
 
-	if(!passed && (D.spread_flags & AIRBORNE) && !internals)
+	if(!passed && (D.spread_flags & AIRBORNE))
```

Comment:
I just want to draw special attention to this fucking line and how messed up it is.